.git-blame-ignore-revs

```diff
@@ -1,2 +1,3 @@
 # Mass reformat:
 24cb8cc878006f3ebf94884102e4e95e2529ac49
```

Comment:
```suggestion
```

Not needed? `git show 24cb8cc878006f3ebf94884102e4e95e2529ac49` shows `fatal: bad object 24cb8cc878006f3ebf94884102e4e95e2529ac49`

Reply:
It's in my local copy, but only on a separate branch, so yeah, this can be dropped